packages/widgetbook/lib/src/settings_panel/settings_panel.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:widgetbook/src/constants/radii.dart';
+import 'package:widgetbook/src/knobs/knobs_panel.dart';
+
+class SettingsPanel extends StatelessWidget {
+  const SettingsPanel({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return DefaultTabController(
+      length: 1,
+      child: Scaffold(
+        appBar: const TabBar(tabs: [
+          Tab(
+            icon: Icon(
+              Icons.build,
+            ),
+          )
+        ]),
```

Comment:
Is scaffold required here?

Reply:
Yea, the tabbar needs to be within the appbar im fairly certain

Reply:
im wrong, I used a column instead and it works fine!